bot/exts/utilities/colour.py

```diff
@@ -78,6 +80,7 @@ async def send_colour_response(self, ctx: commands.Context, rgb: tuple[int, int,
         await ctx.send(file=thumbnail_file, embed=colour_embed)
 
     @commands.group(aliases=("color",), invoke_without_command=True)
+    @whitelist_override(categories=[constants.Categories.development, constants.Categories.media])
```

Comment:
Could you add the details here too? 
https://github.com/python-discord/sir-lancebot/blob/main/bot/__main__.py#L10 
​This decorator completely overrides defaults rather than adding to them.

Reply:
ahhhhh, I will.

Reply:
forgot about this.... fixed: https://github.com/python-discord/sir-lancebot/pull/944/commits/27fae6940e16442d577e1722e81efda5d9c7a9ef